src/main/java/seedu/address/logic/parser/AddTagsCommandParser.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.commands.AddTagsCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Parses input arguments and creates a new AddTagsCommand object
+ */
+public class AddTagsCommandParser implements Parser<AddTagsCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddTagsCommand
+     * and returns an AddTagsCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    @Override
+    public AddTagsCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+
+        if (args.isEmpty() || args.length() < 4) {
```

Comment:
What does these numbers (4 and 3) mean?

Reply:
Oh okay so I was debugging the program and realized that the args passed in is something like " INDEX TAG", so only 4th character onward are the tags.